src/main/resources/resources.properties

```diff
@@ -56,64 +56,64 @@ presentation.dependencyStatus.depError=Error in dependencies
 ## REPL
 ### Java
 ui.repl.configuration.form.chooseWorkingDirectory=Choose Working Directory
-ui.repl.configuration.form.infoLabel=<html><body>This is an A+ Scala REPL configuration window. By default, the working directory and the dependencies, loaded to the REPL classpath belong to the Module, that it was started on. To change the behavior, use the checkbox at the bottom of the window.</body></html>
+ui.repl.configuration.form.infoLabel=<html><body>This dialog sets up the Scala REPL so that the appropriate course modules are automatically available when you use the REPL. Just leave these settings unchanged and click OK below, unless you have a definite reason to do otherwise.</body></html>
 ui.repl.configuration.form.informativeText=Informative text
 ui.repl.configuration.form.workingDirectory=Working directory:
 ui.repl.configuration.form.moduleClasspathSDK=Use classpath and SDK of module:
-ui.repl.configuration.form.doNotShowAgain=Don't show this window again
+ui.repl.configuration.form.doNotShowAgain=Don''t show this window again
```

Comment:
@Aalto-LeTech/idea-plugin-dev need to check if it renders properly, there are some mysterious rules for when the quotation marks are escaped and when not

Reply:
done where possible